schema/mysql/v8/temporal/schema.sql

```diff
@@ -57,7 +57,7 @@ CREATE TABLE current_executions(
   last_write_version BIGINT NOT NULL,
   -- `data` contains the WorkflowExecutionState (same as in `executions.state` above)
   data MEDIUMBLOB NULL,
-  data_encoding VARCHAR(16) NULL,
+  data_encoding VARCHAR(16) NOT NULL DEFAULT '',
```

Comment:
Is this necessary?

Reply:
Wondering if this will cause a backfill on the entire table, if that's the case we should avoid that for this table which could be potentially very large.

Reply:
Both MySQL and PostgreSQL support adding new columns with default values instantly. It's just a table metadata change.

Reply:
> Is this necessary?

Not strictly, but makes the code cleaner.